### ISSUE TITLE
Update vscode extension to 0.0.14

### ISSUE THREAD

**Issue by parlortrickss** [longjoel/ultimate-homebrew#10]
ultimate-homebrew-extensions-0.0.14 is available

I have only checked the GBA deployment, and its currently deploying with v0.0.6

**Comment by longjoel** [comment]
It doesn't always make sense to update. Most of the things are just for the gameboy, like the tile editor. 

Let me know what things would be useful to have in the editor for making gba games, and thanks for opening an issue.

**Comment by parlortrickss** [comment]
Fair enough, i was just pointing out the difference. Not much point updating for GBA if its all GB stuff.
